src/ILVerify/README.md

```diff
@@ -0,0 +1,32 @@
+# ILVerify
+
+### This directory contains the implementation of ILVerify
+
+## Intention of this project:
+The goal is to create a standalone, cross platform, open-source tool, which is capable of verifying MSIL code based on [ECMA-335](https://www.ecma-international.org/publications/standards/Ecma-335.htm).
```

Comment:
Replacing `, which` with `that` would make this better English, I think. (multiple places)

http://www.quickanddirtytips.com/education/grammar/which-versus-that-0